src/seedu/addressbook/ui/OutputFormatter.java

```diff
@@ -0,0 +1,102 @@
+package seedu.addressbook.ui;
+
+import java.util.List;
+
```

Comment:
no header comments; particularly necessary for an abstract class since this is like a contract you provide to the developers. we will talk more about it next week :-)